lib/HLSL/HLOperationLower.cpp

```diff
@@ -878,7 +876,8 @@ Value *TranslatePowImpl(hlsl::OP *hlslOP, IRBuilder<> &Builder, Value *x,
   if (CanUseFxcMulOnlyPatternForPow(Builder, x, y, p)) {
     if (isFXCCompatMode) {
```

Comment:
github won't let me do a suggestion around here, but I think there's several opportunities to remove unnecessary braces here.